ci_tools/parse_pylint_output.py

```diff
@@ -59,6 +114,7 @@
         json.dump(json_data, json_file)
     with open(args.output, mode='a', encoding="utf-8") as md_file:
         md_file.write(output)
+    print(output)
```

Comment:
Do you really want to print the output to terminal? Maybe we can add an option for this?

Reply:
I can remove it, but I tend to think it is a little odd if there is no information in the terminal output of a failing test. This output is human readable.

As an example here:
https://github.com/pyccel/pyccel/actions/runs/7476864741/job/20348250262
The contents of `output` were:
```
# Pylint errors found
## Errors found in module ci_tools/parse_pylint_output.py
-  On line [41](https://github.com/pyccel/pyccel/actions/runs/7476864741/job/20348250262#step:7:42) : Unused variable 'code' (unused-variable)
```
This is what would be removed if the print didn't exist

Reply:
These scripts are only ever really run from the CI as they require a certain amount of effort to set up (e.g. getting the diff with the target branch) so I don't think the output is really critical

Reply:
All right!